src/main/resources/jar_mode_specimin_execution_status.json

```diff
@@ -0,0 +1,23 @@
+{
+  "cf-1291": "FAIL",
```

Comment:
Why are these all FAIL? Is it because specimin-evaluation#37 hasn't yet been merged, so you don't know? Or are they actually all failing in local testing (if so, that's a crisis).

Reply:
Professor, I only configured jar mode for cf-6282, other project is not yet configured for jar mode.
in jar mode, specimin producing minimized program for cf-6282. However compilation is failing  for the minimized program.

Reply:
The printed table will show the reason of failure

Reply:
I'm curious. Did you use the jar files when compiling these projects? In essence, whichever jar files given to Specimin should also be used when compiling the minimized programs.

Reply:
you are right Loi, I did not use the jars when compiling these minimized projects. I need to modify the gradle script to use the jars. Now configuring that.